themes/default/footer.inc

```diff
@@ -1,3 +1 @@
-<p style="text-align: right">
- Created by <a href="http://steve.org.uk/Software/chronicle">Chronicle v<!-- tmpl_var name='chronicle_release' --></a>
-</p>
+<p style="text-align: right">Created by <a href="http://steve.org.uk/Software/chronicle">Chronicle v[% chronicle_release %]</a></p>
```

Comment:
Absolutely. Must have slipped through when I was playing with the templates, sorry!